ansible/roles/azure/tasks/create_vmss.yml

```diff
@@ -40,7 +40,7 @@
   when: numnodes > 100
 
 - name: Create Scale Set
-  azure_rm_virtualmachinescaleset:
+  community.azure.azure_rm_virtualmachinescaleset:
```

Comment:
Hi @arvindshmicrosoft, Is the prefix mandatory? The same module seems to exist in `azure.azcollection` as well.

Reply:
My eventual goal is to support Spot VMSS instances, and that in turn requires this module. Hence thinking forward I figured it might be okay to change this right now.